### ISSUE TITLE
Make the binary app more visible

### ISSUE THREAD

**Issue by abredykhin** [beardedspice/beardedspice#99]
Perhaps I'm too dumb, but it took me 10 minutes to figure out how to get the binary. All I was getting is source code which I couldn't compile due to some errors in cocoa pods

**Comment by afeld** [comment]
Agreed. The homepage is very developer-focused, where it would be great to have a little section at the top around how to install/use it.

**Comment by afeld** [comment]
For example, it took me a few minutes to understand that I needed to click the name of the application in the menu bar drop down to "enable" it.
